src/components/Feedback.js

```diff
@@ -60,7 +60,12 @@ const Feedback = ( { postId, source } ) => {
 					} }
 					className="feedback-button yes"
 				>
-					{ status === 'helpful' && <>&#129395;</> }{ ' ' }
+					{ status === 'helpful' && (
+						<span role="img" aria-label="helpful">
+							{ ' ' }
+							&#129395;{ ' ' }
+						</span>
```

Comment:
Why do we have spacing like this? and not via a CSS Property?

Reply:
Sure will manage the spacing via CSS